install_ffmpeg.sh

```diff
@@ -16,15 +16,13 @@ fi
 if [ ! -e "$HOME/x264/x264" ]; then
   git clone http://git.videolan.org/git/x264.git "$HOME/x264"
   cd "$HOME/x264"
-  # git master as of this writing
-  git checkout 7d0ff22e8c96de126be9d3de4952edd6d1b75a8c
   ./configure --prefix="$HOME/compiled" --enable-pic --enable-static
   make
   make install-lib-static
 fi
 
 if [ ! -e "$HOME/ffmpeg/libavcodec/libavcodec.a" ]; then
-  git clone -b n4.0 https://git.ffmpeg.org/ffmpeg.git "$HOME/ffmpeg" || echo "FFmpeg dir already exists"
+  git clone -b release/4.1 https://git.ffmpeg.org/ffmpeg.git "$HOME/ffmpeg" || echo "FFmpeg dir already exists"
```

Comment:
Same comment as above; let's track tags (which should be immutable) rather than release branches (which can change). There's a `n4.1` tag that we can use.

Reply:
fixed